src/containers/AccountConnection.jsx

```diff
@@ -262,6 +263,12 @@ class AccountConnection extends Component {
               date={account && account.lastSync}
             /> }
 
+            {!success && account && account.auth && <KonnectorFolder
+              connector={connector}
+              account={account}
+              driveUrl={this.store.driveUrl}
```

Comment:
As @gregorylegarec said, it's not a good idea to fetch Drive URL while accounts fetching since it could broke the app. I think you should do the url fetching here from this component since it's not a global data like accounts, what do you think?